rbac/management/group/view.py

```diff
@@ -435,6 +455,7 @@ def update(self, request, *args, **kwargs):
             }
         """
         validate_uuid(kwargs.get("uuid"), "group uuid validation")
+        self.restrict_custom_default_group_renaming(request)
```

Comment:
If you put this after the group = self.get_object(), you can check if the group is custom default group by group.platform_default. Which will simplify your method a lot